tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -89,7 +89,7 @@ describe(__filename, () => {
     sinon.assert.calledWith(fakeTracking.sendEvent, {
       action: variant,
       category: `${AB_HOME_HERO_EXPERIMENT_CATEGORY} / Click`,
-      label: heroItem.props.experimentTitle,
+      label: heroItem.props.children[0].props.children,
```

Comment:
That's a lot of `.` here... Question: can we get access to `sections()` (the method) and from there, get the first data and use those data to "find" components and as expected values?

Reply:
The sections method returns the react component but I think if move the heroes outside of this function and call that in the test here something like that could work. If that's better I'll update..

Reply:
I don't know if it's much better in that it's almost as long but let me know what you think..

Reply:
what do you think?